lib/semantics/check-omp-structure.cc

```diff
@@ -56,6 +56,17 @@ static constexpr OmpDirectiveSet targetSet{OmpDirective::TARGET,
     OmpDirective::TARGET_TEAMS_DISTRIBUTE_PARALLEL_DO_SIMD,
     OmpDirective::TARGET_TEAMS_DISTRIBUTE_SIMD};
 
+static constexpr inline OmpClauseSet targetAllowedClauses{OmpClause::IF,
```

Comment:
They're already sets of clauses, I think we can drop the `clauses` from variable name: `targetAllowed`.

Reply:
I prefer the name to says what it is, so I prefer targetAllowedClauses.

Reply:
Why are these variables declared `inline`?

Reply:
I wanted to signal that I didn't really want these variables to be present in the generated binary, however after checking the standard I realise a) that's not how inline variables work and b) constexpr implies inline anyway. I'll remove this when I do the actual implementation.